Makefile-lib.am

```diff
@@ -23,6 +23,7 @@ lib_LTLIBRARIES += librpmostree-1.la
 
 librpmostreeincludedir = $(includedir)/rpm-ostree-1
 librpmostreeinclude_HEADERS = $(librpmostree_public_headers)
+GITIGNOREFILES += src/lib/rpmostree-version.h
```

Comment:
Hmm, this should get picked up automatically no? Did you try deleting & regenerating your `.gitignore` file? I had to do that once in order to get the changes.